### PR TITLE
Increase dump limit and remove extension from filename

### PR DESCRIPTION
**Pull request by pwnbus**

### CODE REVIEW

File: cmd/iprepd/main.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func IPBlocklistGCS(config iprepd.ServerCfg, reputationDump []iprepd.Reputation)
 		ipBlocklistContents = ipBlocklistContents + fmt.Sprintf("%s/32\n", rep.Object)
 
 		cnt++
-		if cnt == 512 {
+		if cnt == 20000 {
```
Comment (pwnbus) [inline cmd/iprepd/main.go:77]: After testing with netops, looks like 20k shouldn't be a problem here.

File: cmd/iprepd/main.go [hunk 2]
```diff
@@ -58,7 +58,7 @@ func main() {
 
 func IPBlocklistGCS(config iprepd.ServerCfg, reputationDump []iprepd.Reputation) error {
 	var (
-		blocklistFile       = "./ip-blocklist.txt"
+		blocklistFile       = "./ip-blocklist"
```
Comment (pwnbus) [inline cmd/iprepd/main.go:61]: So this is really dumb, but the firewalls would complain and error out if the extension was .txt (you can't make this stuff up). Simply removing the extension, caused the error to go away, and everything to start working :\

File: cmd/iprepd/main.go (hunk 1 above)
Comment (pwnbus) [reply cmd/iprepd/main.go]: Moved this to a config option.

File: cmd/iprepd/main.go (hunk 2 above)
Comment (kkleemola) [reply cmd/iprepd/main.go:61]: This is just the local file, the path it gets uploaded to is in the config. This is fine to change but its the config change that will do what you need.

**Review by kkleemola — APPROVED** [review]

**Pull request merged**